tavern/schemas/extensions.py

```diff
@@ -13,6 +14,23 @@
 logger = logging.getLogger(__name__)
 
 
+def vaidate_last(func):
+    """Decorator to execute custom validator after all nested validation will
+    be passed."""
+    @functools.wraps(func)
+    def wrapper(value, rule_obj, path):
+        schema = rule_obj.schema
+        if schema is not None:
+            schema = schema.copy()
+            del schema['func']
+            c = core.Core(source_data=value, schema_data=schema)
+            c.validate(raise_exception=True)
+        else:
+            print(value, rule_obj, path)
```

Comment:
Is this print here for a reason? It should be removed, raise an exception, or changed to a log message if it is useful in some way